source/codegen/metadata/niswitch/config.py

```diff
@@ -1,30 +1,20 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-SWITCH API metadata version 19.6.0d7
+# -*- coding: utf-8 -*-
+# This file is generated from NI-SWITCH API metadata version 22.8.0d9999
 config = {
     'additional_headers': {
         'custom/ivi_errors.h': [
             'service.cpp'
         ]
     },
-    'api_version': '19.6.0d7',
+    'api_version': '22.8.0d9999',
```

Comment:
Oh yeah, this is what I was talking about earlier in person. This and the version printed in the comment above all these files. That's mainly why we don't want to submit this version I think. I think it's fine once the ni-central PR goes in and we update from the metadata in that export. Then we'll get an actual export version here for the api_version.

We also haven't talked about it but in my mind for all these MI drivers we can do updates to development versions of the API between grpc-device releases. But when we prepare a new release, we'll pull the metadata from a final export in ni-central.